modules/living_spaces_intranet/living_spaces_intranet.install

```diff
@@ -94,3 +94,31 @@ function living_spaces_intranet_update_9001() {
     }
   }
 }
+
+/**
+ * Update default "allow_admins_to_edit_members" value.
+ */
+function living_spaces_intranet_update_9002() {
+  $uuid_list = [
+    'c7ecd1bf-9377-49cc-b0fd-a06bb246847a',
+    '5ab99cc1-45b7-45ec-a9b4-865e7958e553',
+    '6c8f2d09-1147-43b5-ab2e-cea435e33ad7',
```

Comment:
1. Why do you update the same groups two times in the profile and dip_counseling module?
2. We don't have a group with `6c8f2d09-1147-43b5-ab2e-cea435e33ad7` uuid in the profile.

Reply:
Done.